lldb/source/Symbol/TypeSystem.cpp

```diff
@@ -299,29 +299,38 @@ llvm::Expected<TypeSystem &> TypeSystemMap::GetTypeSystemForLanguage(
 
 llvm::Expected<TypeSystem &>
 TypeSystemMap::GetTypeSystemForLanguage(lldb::LanguageType language,
-                                        Target *target, bool can_create,
-// BEGIN: SWIFT
-                                        const char *compiler_options) {
-// END: SWIFT
+                                        Module *module, bool can_create) {
   if (can_create) {
     return GetTypeSystemForLanguage(
-        language, llvm::Optional<CreateCallback>([language, target,
-// BEGIN SWIFT
-                                                  compiler_options]() {
-          return TypeSystem::CreateInstance(language, target, compiler_options);
-// END SWIFT
+        language, llvm::Optional<CreateCallback>([language, module]() {
+          return TypeSystem::CreateInstance(language, module);
         }));
   }
   return GetTypeSystemForLanguage(language);
 }
 
 llvm::Expected<TypeSystem &>
 TypeSystemMap::GetTypeSystemForLanguage(lldb::LanguageType language,
-                                        Module *module, bool can_create) {
+                                        Target *target, bool can_create) {
   if (can_create) {
     return GetTypeSystemForLanguage(
-        language, llvm::Optional<CreateCallback>([language, module]() {
-          return TypeSystem::CreateInstance(language, module);
+        language, llvm::Optional<CreateCallback>([language, target]() {
+          return TypeSystem::CreateInstance(language, target);
+        }));
+  }
+  return GetTypeSystemForLanguage(language);
+}
+
+// BEGIN APPLE
+llvm::Expected<TypeSystem &>
+TypeSystemMap::GetTypeSystemForLanguage(lldb::LanguageType language,
```

Comment:
I don't think the compiler options ever get used in the swift code anymore, so we don't really need this version either.  It won't hurt anything to leave it in for now, but we should remove it at some point.